### ISSUE TITLE
Entry Editor won't run

### ISSUE THREAD

**Issue by riverBanjo** [CalderaWP/Caldera-Forms#3666]
## What Version Of Caldera Forms, WordPress and PHP Are You Using?
* WordPress Version: 5.6PHP Version: 7.4.1MySQL Version: 8.0.16Caldera Forms Version: 1.9.4-b1

## What Is The Unexpected Behaviour?
* You can't edit an entry with Caldera Forms.

## What PHP Errors Have You Logged While Reproducing This Bug?

[06-Jan-2021 22:24:30 UTC] PHP Warning:  file_exists() expects parameter 1 to be a valid path, array given in C:\Users\River\Local Sites\wp56\app\public\wp-content\plugins\1.9.4-b1\cf2\Jobs\DeleteFileJob.php on line 41
[06-Jan-2021 22:24:30 UTC] PHP Stack trace:
[06-Jan-2021 22:24:30 UTC] PHP   1. {main}() C:\Users\River\Local Sites\wp56\app\public\wp-cron.php:0
[06-Jan-2021 22:24:30 UTC] PHP   2. do_action_ref_array() C:\Users\River\Local Sites\wp56\app\public\wp-cron.php:138
[06-Jan-2021 22:24:30 UTC] PHP   3. WP_Hook->do_action() C:\Users\River\Local Sites\wp56\app\public\wp-includes\plugin.php:551
[06-Jan-2021 22:24:30 UTC] PHP   4. WP_Hook->apply_filters() C:\Users\River\Local Sites\wp56\app\public\wp-includes\class-wp-hook.php:311
[06-Jan-2021 22:24:30 UTC] PHP   5. WP_Queue\Cron->cron_worker() C:\Users\River\Local Sites\wp56\app\public\wp-includes\class-wp-hook.php:287
[06-Jan-2021 22:24:30 UTC] PHP   6. WP_Queue\Worker->process() C:\Users\River\Local Sites\wp56\app\public\wp-content\plugins\1.9.4-b1\vendor\a5hleyrich\wp-queue\src\WP_Queue\Cron.php:105
[06-Jan-2021 22:24:30 UTC] PHP   7. calderawp\calderaforms\cf2\Jobs\DeleteFileJob->handle() C:\Users\River\Local Sites\wp56\app\public\wp-content\plugins\1.9.4-b1\vendor\a5hleyrich\wp-queue\src\WP_Queue\Worker.php:47
[06-Jan-2021 22:24:30 UTC] PHP   8. file_exists() C:\Users\River\Local Sites\wp56\app\public\wp-content\plugins\1.9.4-b1\cf2\Jobs\DeleteFileJob.php:41
[06-Jan-2021 22:24:30 UTC] PHP Warning:  dirname() expects parameter 1 to be string, array given in C:\Users\River\Local Sites\wp56\app\public\wp-content\plugins\1.9.4-b1\cf2\Jobs\DeleteFileJob.php on line 78
[06-Jan-2021 22:24:30 UTC] PHP Stack trace:
[06-Jan-2021 22:24:30 UTC] PHP   1. {main}() C:\Users\River\Local Sites\wp56\app\public\wp-cron.php:0
[06-Jan-2021 22:24:30 UTC] PHP   2. do_action_ref_array() C:\Users\River\Local Sites\wp56\app\public\wp-cron.php:138
[06-Jan-2021 22:24:30 UTC] PHP   3. WP_Hook->do_action() C:\Users\River\Local Sites\wp56\app\public\wp-includes\plugin.php:551
[06-Jan-2021 22:24:30 UTC] PHP   4. WP_Hook->apply_filters() C:\Users\River\Local Sites\wp56\app\public\wp-includes\class-wp-hook.php:311
[06-Jan-2021 22:24:30 UTC] PHP   5. WP_Queue\Cron->cron_worker() C:\Users\River\Local Sites\wp56\app\public\wp-includes\class-wp-hook.php:287
[06-Jan-2021 22:24:30 UTC] PHP   6. WP_Queue\Worker->process() C:\Users\River\Local Sites\wp56\app\public\wp-content\plugins\1.9.4-b1\vendor\a5hleyrich\wp-queue\src\WP_Queue\Cron.php:105
[06-Jan-2021 22:24:30 UTC] PHP   7. calderawp\calderaforms\cf2\Jobs\DeleteFileJob->handle() C:\Users\River\Local Sites\wp56\app\public\wp-content\plugins\1.9.4-b1\vendor\a5hleyrich\wp-queue\src\WP_Queue\Worker.php:47
[06-Jan-2021 22:24:30 UTC] PHP   8. calderawp\calderaforms\cf2\Jobs\DeleteFileJob->dirName() C:\Users\River\Local Sites\wp56\app\public\wp-content\plugins\1.9.4-b1\cf2\Jobs\DeleteFileJob.php:45
[06-Jan-2021 22:24:30 UTC] PHP   9. dirname() C:\Users\River\Local Sites\wp56\app\public\wp-content\plugins\1.9.4-b1\cf2\Jobs\DeleteFileJob.php:78
[06-Jan-2021 22:29:26 UTC] PHP Warning:  file_exists() expects parameter 1 to be a valid path, array given in C:\Users\River\Local Sites\wp56\app\public\wp-content\plugins\1.9.4-b1\cf2\Jobs\DeleteFileJob.php on line 41
[06-Jan-2021 22:29:26 UTC] PHP Stack trace:
[06-Jan-2021 22:29:26 UTC] PHP   1. {main}() C:\Users\River\Local Sites\wp56\app\public\wp-cron.php:0
[06-Jan-2021 22:29:26 UTC] PHP   2. do_action_ref_array() C:\Users\River\Local Sites\wp56\app\public\wp-cron.php:138
[06-Jan-2021 22:29:26 UTC] PHP   3. WP_Hook->do_action() C:\Users\River\Local Sites\wp56\app\public\wp-includes\plugin.php:551
[06-Jan-2021 22:29:26 UTC] PHP   4. WP_Hook->apply_filters() C:\Users\River\Local Sites\wp56\app\public\wp-includes\class-wp-hook.php:311
[06-Jan-2021 22:29:26 UTC] PHP   5. WP_Queue\Cron->cron_worker() C:\Users\River\Local Sites\wp56\app\public\wp-includes\class-wp-hook.php:287
[06-Jan-2021 22:29:26 UTC] PHP   6. WP_Queue\Worker->process() C:\Users\River\Local Sites\wp56\app\public\wp-content\plugins\1.9.4-b1\vendor\a5hleyrich\wp-queue\src\WP_Queue\Cron.php:105
[06-Jan-2021 22:29:26 UTC] PHP   7. calderawp\calderaforms\cf2\Jobs\DeleteFileJob->handle() C:\Users\River\Local Sites\wp56\app\public\wp-content\plugins\1.9.4-b1\vendor\a5hleyrich\wp-queue\src\WP_Queue\Worker.php:47
[06-Jan-2021 22:29:26 UTC] PHP   8. file_exists() C:\Users\River\Local Sites\wp56\app\public\wp-content\plugins\1.9.4-b1\cf2\Jobs\DeleteFileJob.php:41
[06-Jan-2021 22:29:26 UTC] PHP Warning:  dirname() expects parameter 1 to be string, array given in C:\Users\River\Local Sites\wp56\app\public\wp-content\plugins\1.9.4-b1\cf2\Jobs\DeleteFileJob.php on line 78
[06-Jan-2021 22:29:26 UTC] PHP Stack trace:
[06-Jan-2021 22:29:26 UTC] PHP   1. {main}() C:\Users\River\Local Sites\wp56\app\public\wp-cron.php:0
[06-Jan-2021 22:29:26 UTC] PHP   2. do_action_ref_array() C:\Users\River\Local Sites\wp56\app\public\wp-cron.php:138
[06-Jan-2021 22:29:26 UTC] PHP   3. WP_Hook->do_action() C:\Users\River\Local Sites\wp56\app\public\wp-includes\plugin.php:551
[06-Jan-2021 22:29:26 UTC] PHP   4. WP_Hook->apply_filters() C:\Users\River\Local Sites\wp56\app\public\wp-includes\class-wp-hook.php:311
[06-Jan-2021 22:29:26 UTC] PHP   5. WP_Queue\Cron->cron_worker() C:\Users\River\Local Sites\wp56\app\public\wp-includes\class-wp-hook.php:287
[06-Jan-2021 22:29:26 UTC] PHP   6. WP_Queue\Worker->process() C:\Users\River\Local Sites\wp56\app\public\wp-content\plugins\1.9.4-b1\vendor\a5hleyrich\wp-queue\src\WP_Queue\Cron.php:105
[06-Jan-2021 22:29:26 UTC] PHP   7. calderawp\calderaforms\cf2\Jobs\DeleteFileJob->handle() C:\Users\River\Local Sites\wp56\app\public\wp-content\plugins\1.9.4-b1\vendor\a5hleyrich\wp-queue\src\WP_Queue\Worker.php:47
[06-Jan-2021 22:29:26 UTC] PHP   8. calderawp\calderaforms\cf2\Jobs\DeleteFileJob->dirName() C:\Users\River\Local Sites\wp56\app\public\wp-content\plugins\1.9.4-b1\cf2\Jobs\DeleteFileJob.php:45
[06-Jan-2021 22:29:26 UTC] PHP   9. dirname() C:\Users\River\Local Sites\wp56\app\public\wp-content\plugins\1.9.4-b1\cf2\Jobs\DeleteFileJob.php:78

## What JavaScript Errors Have You Seen While Reproducing This Bug?

![2021-01-05_13-44-00](https://user-images.githubusercontent.com/43787368/103910285-a8c88580-50d2-11eb-8645-ed07295ae3da.png)

**Comment by twomice** [comment]
Me too. Seeing the same behavior in WordPress 5.6, CF 1.9.3. 

To be more specific:

- After clicking the View button for an entry, the entry viewer is displayed correctly.
- After clicking the Edit Entry button in the entry viewer, one of two things may happen:
  - The spinning "loading" indicator will display, and things appear to stall here; i.e., the "edit" form is  never loaded
  - The "edit" form will display, but the "View Entry" and "Save Changes" buttons are disabled.
- In either case, this javascript error appears in the console (same as shown in the OP's last image):
```
admin.php?page=caldera-forms:1704 Uncaught TypeError: form.parsley is not a function
    at Object.callback (admin.php?page=caldera-forms:1704)
    at Object.target (wp-baldrick-full.min.js?ver=1.9.3:2)
    at c (wp-baldrick-full.min.js?ver=1.9.3:2)
    at Object.success (wp-baldrick-full.min.js?ver=1.9.3:3)
    at i (jquery-12-4.min.js?ver=5.6:2)
    at Object.fireWith [as resolveWith] (jquery-12-4.min.js?ver=5.6:2)
    at y (jquery-12-4.min.js?ver=5.6:4)
    at XMLHttpRequest.c (jquery-12-4.min.js?ver=5.6:4)
```

**Comment by New0** [comment]
This issue should be fixed in 1.9.4 - https://downloads.wordpress.org/plugin/caldera-forms.1.9.4.zip - don't hesitate to open this issue again if this wasn't the case.

Thank you all for reporting this issue and for your patience.
